Protecs/src/test/java/uk/ac/ucl/protecs/helperFunctions/helperFunctions.java

```diff
@@ -129,6 +144,70 @@ public static void SetFractionObjectsWithCertainBehaviourNode(double fraction, W
 	}
 	}
 	
+	public static int GetNumberAlive(WorldBankCovid19Sim world) {
+		int counter = 0;
+		for (Person p: world.agents) {if (p.isAlive()) {counter++;}}
+		return counter;
+	}
+
+	public static void HaltDiseaseProgressionAtStage(WorldBankCovid19Sim world, String stage) {
+		// You present this function with a stage in the disease which you want to halt the infection, then this
+		// function changes the parameters which allows the disease to progress further
+		switch (stage) {
+		case "Exposed":
+			int exp_idx = 0;
+			// Make sure there are no transitions from exposed to symptomatic COVID
+			for (double val: world.params.infection_p_sym_by_age) {
```

Comment:
@swise5 to review this for her own sanity because this looks right but I'm twitchy